.readthedocs.yaml

```
@@ -2,7 +2,7 @@ version: 2
 build:
   os: ubuntu-20.04
   tools:
-    python: "3.8"
+    python: "3.12"
```

Let's just use the current Python so we can spot errors with the current Python as they arise.
```suggestion
    python: "3.x"
```